### PR TITLE
Fix unmount error from react-tether

### PR DESCRIPTION
**Pull request by rafeememon**
Re: #357 

Right now this is just a failing unit test which should pass when we get a fixed version of react-tether.

### CODE REVIEW

File: src/datepicker.jsx [hunk 1]
```diff
@@ -165,7 +165,9 @@ var DatePicker = React.createClass({
             open={this.state.open} />
           {this.renderClearButton()}
         </div>
-        {this.renderCalendar()}
+        <div>
```
Comment (rafeememon) [inline src/datepicker.jsx]: :sob: This fixes the immediate issue

**Comment by rafeememon** [comment]
@martijnrusschen could you cut a patch release that includes this fix?

**Comment by rafeememon** [comment]
Actually it needs to be a real release because of the moment refactor

**Comment by martijnrusschen** [comment]
Yep, will kick off a minor release. 1.0 is coming closer!

**Comment by rafeememon** [comment]
:+1:

**Comment by martijnrusschen** [comment]
https://github.com/Hacker0x01/react-datepicker/releases/tag/v0.23.0

**Pull request merged**